tensorflow/training/docker/2.5/py3/Dockerfile.cpu

```diff
@@ -118,29 +117,13 @@ RUN ${PIP} --no-cache-dir install --upgrade \
     pip \
     setuptools
 
```

Comment:
removed openssl installation from source in favor of apt installation

Reply:
I don't see the installation of openssl from apt in the dockerfile.